bindings/python/.evergreen/test.sh

```diff
@@ -34,6 +38,10 @@ elif [ "Darwin" = "$(uname -s)" ]; then
              "/Library/Frameworks/Python.framework/Versions/3.8/bin/python3"
              "/Library/Frameworks/Python.framework/Versions/3.9/bin/python3"
              "/Library/Frameworks/Python.framework/Versions/3.10/bin/python3")
+    sudo ${PYTHONS[${#PYTHONS[@]}-3]} fix_ssl.py || true
```

Comment:
We need this because otherwise we get weird SSL errors like this: https://evergreen.mongodb.com/task/libmongocrypt_macos_x86_64_test_python_patch_d0c748a2a249c5ce2b70485cc05fd4e5f7b152c0_6286d6030ae60669492cecc5_22_05_19_23_43_22

Reply:
And we need sudo to do the permissions change.